packages/analytics-plugin-segment/src/browser.js

```diff
@@ -122,8 +122,9 @@ function segmentPlugin(pluginConfig = {}) {
       if (typeof analytics === 'undefined') return
       const properties = payload.properties || {}
       const name = properties.name || properties.title
+      const category = properties.category
       
-      analytics.page(name, properties, {
+      analytics.page(category, name, properties, {
```

Comment:
This looks like it might be a breaking change.

Can you verify that if `category` is null that `name` still works here?

Reply:
Hi David,
We did check this, segment do some argument handling on their end, and it works. 
[See here](https://github.com/segmentio/analytics.js-core/blob/master/lib/analytics.ts#L565)

Reply:
Thank you for verifying! Will get this released shortly

Reply:
Hi David, the PR build is breaking. @analytics/queue-utils failed build on this package. How can I fix this? Thanks